config/examples/core.yml

```diff
@@ -1,6 +1,6 @@
 base: &default
-  url: <%= ENV.fetch('THREESCALE_CORE_INTERNAL_API', 'http://localhost:3001/internal/') %>
-  fake_server: 'http://localhost:3000/internal/'
+  url: "<%= URI.join(ENV.fetch('BACKEND_URL', 'http://localhost:3001'), '/internal/') %>"
```

Comment:
See the comment here https://github.com/3scale/porta/pull/3107/files#r1024235636

The proposal is to have `BACKEND_URL` (or `BACKEND_INTERNAL_URL` ) env var, and we can append `/internal/` here.